CHANGELOG.md

```diff
@@ -4,6 +4,24 @@ All notable changes to this project will be documented in this file.
 Each new release typically also includes the latest modulesync defaults.
 These should not affect the functionality of the module.
 
+## [v4.1.1](https://github.com/voxpupuli/puppet-snmp/tree/v4.1.1) (2019-02-07)
```

Comment:
should this be 5.0.0 because we have breaking changes listed?

Reply:
what breaking changes?